src/TypeScript.UnitTests/Rules/ActiveRulesCalculatorTests.cs

```diff
@@ -23,29 +23,17 @@
 using System.Linq;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
 using SonarLint.VisualStudio.Core;
-using SonarLint.VisualStudio.Integration.UnitTests;
 using SonarLint.VisualStudio.TypeScript.EslintBridgeClient.Contract;
 using SonarLint.VisualStudio.TypeScript.Rules;
 
 namespace SonarLint.VisualStudio.TypeScript.UnitTests.Rules
 {
     [TestClass]
-    public class ActiveJavaScriptRulesProviderTests
+    public class ActiveRulesCalculatorTests
```

Comment:
These tests now cover the helper class.
The only real change is to remove the MEF test.